gnosis/safe-react-apps
# [Compound] Get markets config from REST api

## What it solves
Resolves #223, resolves #270

## How this PR fixes it
We are retrieving the markets now from the Compound REST api. It only works on mainnet. For Rinkeby we fallback to some coins for testing purposes

## How to test it
1) Open Compound app (mainnet) and connect to your safe from https://app.compound.finance using WalletConnect
2) Check the market selector show the same markets as in the Compound web app. Order should be the same
3) Go market by market and check the Supply and Distribution APYs against the web app
4) General features from compound safe app should remain the same
